docs-src/prod-readiness-context/how-to-set-up-codec-server.md

```diff
@@ -23,7 +24,9 @@ Note that when you use a Codec Server, the decoded payloads are visible only to
 Because you create, operate, and manage access to your Codec Server in your controlled environment, ensure that you consider the following:
 
 - When you set your codec endpoint with your Web UI, expect your Codec Server to receive a large number of requests per Workflow Execution from the Web UI.
-- Ensure that you secure access to the decrypted data from your Codec Server. <!--Need a better way to explain this; with temporal cloud, the decrypted data is sent to the browser; there is no guarantee that the cloud ui is hosted in a particualr region etc.> need clearer way to set this expectation.-->
+- Ensure that you secure access your Codec Server.
+  For details, see [Authorization](#authorization).
+  <!--Need a better way to explain this; with temporal cloud, the decrypted data is sent to the browser; there is no guarantee that the cloud ui is hosted in a particular region etc.> need clearer way to set this expectation.-->
```

Comment:
Do we need to address the commented line <!-- Need a better way ...> as a dependency to publish the docs for the new Codec Server endpoint per Namespace?

Reply:
This is not a blocker.